ts/routes/graphs/RangeBox.svelte

```diff
@@ -83,6 +83,8 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
             {collection}
         </label>
 
+        <!-- This form is an external API and should not be changed -
+	other clients e.g. AnkiDroid programmatically update this form -->
         <input
             type="text"
```

Comment:
@dae - I looked on svelte and I *think* this is how we add an id? And if so, I believe it's a reasonable id?
This would unblock the cross-repo PR thing we have going here, I'm just making a best effort to move it forward :-)

```suggestion
            type="text"
            id="searchText"
```